cmake/modules/FindProtobuf.cmake

```diff
@@ -398,6 +398,7 @@ find_program(Protobuf_PROTOC_EXECUTABLE
     PATHS
     ${Protobuf_SRC_ROOT_FOLDER}/vsprojects/${_PROTOBUF_ARCH_DIR}Release
     ${Protobuf_SRC_ROOT_FOLDER}/vsprojects/${_PROTOBUF_ARCH_DIR}Debug
+    NO_CMAKE_SYSTEM_PATH
```

Comment:
Will this work for x86_64 version without an explicit path/hints?

Reply:
I intended to turn off searching in sysroot first, but will check it again.